contributor-docs/testing.md

```diff
@@ -132,6 +132,9 @@ available, check out the table below.
 > However, screenshots will not match and new ones will need to be generated on
 > your first test run.
 
+> [!TIP]
```

Comment:
Documenting this awesome pro-tip that Josh informed me about!

Reply:
Thank you for adding this!